python_utils.py

```diff
@@ -284,6 +284,11 @@ def url_retrieve(source_url, filename=None):
     """
     try:
         import urllib
+        # Change the User-Agent to prevent servers from blocking requests.
+        # See https://support.cloudflare.com/hc/en-us/articles/360029779472-Troubleshooting-Cloudflare-1XXX-errors#error1010. # pylint: disable=line-too-long
```

Comment:
Do we need to pragma here? (links can expand to more than 80 chars*) [not 100% sure!]

Reply:
No, I don't think so. It errored for me when I ran the pre_commit_linter.